pkgs/development/node-packages/node-env.nix

```diff
@@ -112,47 +112,40 @@ let
     ''
       DIR=$(pwd)
       cd $TMPDIR
-
+      
       unpackFile ${src}
-
+      
       # Make the base dir in which the target dependency resides first
       mkdir -p "$(dirname "$DIR/${packageName}")"
 
       if [ -f "${src}" ]
       then
           # Figure out what directory has been unpacked
-          packageDir="$(find . -maxdepth 1 -type d | tail -1)"
-
+          packageDir=$(find . -type d -maxdepth 1 | tail -1)
+          
           # Restore write permissions to make building work
           find "$packageDir" -type d -print0 | xargs -0 chmod u+x
           chmod -R u+w "$packageDir"
-
+          
           # Move the extracted tarball into the output folder
           mv "$packageDir" "$DIR/${packageName}"
       elif [ -d "${src}" ]
       then
```

Comment:
I only added one line in  `pkgs/top-level/node-packages.json`, the other changes are auto generated by a script, invoked to the letter of the instructions AFAIK.